tools/branch_comparisons.md

```diff
@@ -0,0 +1,106 @@
+# Branch comparisons
+
+These links take to comparisons between branches.
+
+If a cell in the table is `-`, this means that the two Ignition releases use
+the same version of this Ignition library. So, for example, `ign-cmake` has a
+`-` for B ➡️  C because Blueprint and Citadel use `ign-cmake2`.
+
+When it says "There isn’t anything to compare," this means no pull requests
+need to be forward-ported.
+
+Otherwise, those branches could use a forward-port pull request.
+See the instructions on the
+[contributing guide](https://ignitionrobotics.org/docs/all/contributing#process).
+
+* **B**: Blueprint
+* **C**: Citadel
+* **D**: Dome
+* **main**: main branch
+
+> SDFormat is a special case where the release branches should be compared
+  against `merge_N_M_reference` branches instead of other release branches.
+  Be sure to update those branches before using for comparison.
```

Comment:
Do we still need this if SDFormat is following the Ignition porting strategy now?

Reply:
a667d98